lm_eval/models/gpt2.py

```diff
@@ -3,6 +3,14 @@
 from lm_eval.base import BaseLM
 
 
+def get_args_for_accelerate(pretrained, max_memory):
+    config = transformers.AutoConfig.from_pretrained(pretrained)
+    max_memory = {i:max_memory for i in range(torch.cuda.device_count())}
```

Comment:
There should be the CPU max memory here too in case there are not enough GPUs, to first offload on the CPU and then offload on disk only when the CPU RAM is full. You can look how it's computed in the Accelerate code base.

Reply:
Yeah indeed and it moved directly the model in disk when the cpu is not specified. Added a new arg in 30b6b1bd54b0f406507cc1cfac69e76f4a0551f2